__tests__/spec/sanity-checks.js

```diff
@@ -135,9 +140,9 @@ describe('The Prototype Kit', () => {
     })
   })
 
-  const sassFiles = glob.sync(gulpConfig.paths.assets + '/sass/*.scss')
+  const sassFiles = glob.sync(buildConfig.paths.assets + '/sass/*.scss')
 
-  describe(`${gulpConfig.paths.assets}sass/`, () => {
+  describe(`${buildConfig.paths.assets}sass/`, () => {
     it.each(sassFiles)('%s renders to CSS without errors', async (file) => {
       return new Promise((resolve, reject) => {
         sass.render({
```

Comment:
I wonder if we can change this line to use a function from `build/run-tasks.js`... that feels like it would be a better test?